### PR TITLE
Success story, integration item, and blog post for the Lombiq UI Testing Toolbox for Orchard Core

### PR DESCRIPTION
**Pull request by Piedone**
Added a success story, integration item, and blog post for the [Lombiq UI Testing Toolbox for Orchard Core](https://github.com/Lombiq/UI-Testing-Toolbox) that can run ZAP scans in UI tests. See its docs [here](https://github.com/Lombiq/UI-Testing-Toolbox/blob/dev/Lombiq.Tests.UI/Docs/SecurityScanning.md).

### CODE REVIEW

**Review by psiinon — APPROVED** [review]
Thanks @Piedone - this is great!
I assume you are ok with the DCO?

File: site/content/blog/2023-12-08-automated-zap-scans-for-orchard-core-apps/index.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+[Orchard Core](https://orchardcore.net/) is an open source ASP.NET Core web framework and Content Management System. It's a great one, as well, but due to spending the last 12 years with it, I may be a bit biased. Anyway, if you're a .NET web developer, I highly recommend you check it out so you don't need to reinvent the wheel in your own projects for things like content/user/media management, audit trail, workflows, and much more.
+
+So, where do ZAP and Orchard Core meet? Now, as part of the open source [Lombiq UI Testing Toolbox for Orchard Core](https://github.com/Lombiq/UI-Testing-Toolbox) you can run ZAP scans for Orchard Core apps, right in a UI test suite, simply by using the .NET configuration API or completely custom Automation Framework plans even.
+
+## Running ZAP from a UI Test
+
+This is what such a ZAP scan-running UI test looks like, written in C#, using the [xUnit](https://xunit.net/) testing framework:
+
+```java
```
Comment (Piedone) [inline site/content/blog/2023-12-08-automated-zap-scans-for-orchard-core-apps/index.md:30]: The code is C#, but it seems that syntax highlighting for that is not working, though [it's supported by Hugo](https://gohugo.io/content-management/syntax-highlighting/#list-of-chroma-highlighting-languages).

Comment (Piedone) [reply site/content/blog/2023-12-08-automated-zap-scans-for-orchard-core-apps/index.md:30]: That's what I tried (and `c#`) but neither resulted in the code block getting syntax highlighting (the text just stayed all white).

Comment (kingthorin) [reply site/content/blog/2023-12-08-automated-zap-scans-for-orchard-core-apps/index.md:30]: Are you actually building/deploying locally?

Comment (Piedone) [reply site/content/blog/2023-12-08-automated-zap-scans-for-orchard-core-apps/index.md:30]: Yes, I've run the site, and hence why I see it not working.

Comment (kingthorin) [reply site/content/blog/2023-12-08-automated-zap-scans-for-orchard-core-apps/index.md:30]: Okay was just trying to make sure it was live and not you trusting GitHub's rendering 😉

**Review by thc202 — APPROVED** [review]

**Comment by Piedone** [comment]
Thank you for ZAP!

**Comment by psiinon** [comment]
Will do, but the blog post is dated for tomorrow, so we shouldnt merge it until then 😉

**Review by psiinon — APPROVED** [review]

**Comment by Piedone** [comment]
I won't have too much time tomorrow, so if you can, please provide any feedback until then (or if it's just trivial like typos then please don't wait for me).

**Pull request merged**